target/linux/ramips/dts/mt7628an_wavlink_wl-wn577a2.dts

```diff
@@ -0,0 +1,117 @@
+/dts-v1/;
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+#include "mt7628an.dtsi"
+
+/ {
+	compatible = "wavlink,wl-wn577a2", "maginon,wlr-755", "mediatek,mt7628an-soc";
+	model = "WAVLINK WL-WN577A2";
+
+	chosen {
+		bootargs = "console=ttyS0,57600";
```

Comment:
That's already in mt7628an.dtsi ...

Reply:
Ok, I'll remove it. My dts is based on mt7628an_wavlink_wl-wn575a3.dts...